README.md

```diff
@@ -172,3 +172,14 @@ Functional tests are written with PHPUnit. Note that Bundles and Components are
 ## Remove demo configuration
 
 If you start a project from the sandbox, remove ```.sensiolabs.yml``` as its not a good example for production use.
+
+## Apply vagrant box
+
+There's a new vagrant box that is simple to boot:
+
+    vagrant up
+
+Now you can access the cmf on the browser:
+
+    cmf.lo
```

Comment:
lets write http://cmf.lo/app_dev.php - do you also clear the prod cache in the vagrant setup? even then app_dev.php makes probably more sense